bentoml/frameworks/spark_mllib.py

```diff
@@ -0,0 +1,92 @@
+import importlib
+import json
+import logging
+import os
+
+from bentoml.exceptions import BentoMLException, MissingDependencyException
+from bentoml.service import BentoServiceArtifact, BentoServiceEnv
+
+
+class PySparkModelArtifact(BentoServiceArtifact):
```

Comment:
@akainth015 could you add example usage here in the docstring of this class? and would be great to add it to the supported frameworks list in the docs https://docs.bentoml.org/en/latest/frameworks.html, you can also find some other example docs for other Artifact classes there

Reply:
Done! I've added a docstring